command_handler.py

```diff
@@ -1,23 +1,34 @@
 import logging
-from aiogram import types
+from aiogram import types, Bot
 from aiogram.types import ParseMode
 import config
 
 logger = logging.getLogger(__name__)
-HELP = config.help_text
+ABOUT_HTML = config.ABOUT_HTML
+HELP_HTML = config.HELP_HTML
 
 
 class CommandHandler:
+
+    def __init__(self, bot: Bot):
```

Comment:
passing the entire bot object here just because we need a username looks unnecessary, what i would suggest instead is to  pass the username in `handle` method. So this way your call of `handle` method from main.py will be 
```
username = (await self.bot.get_me()).username)
await commandHandler.handle(message, username)
```

Reply:
> `username = (await self.bot.get_me()).username`

don't know how to do this in main.py, the `username =` is not under a class, `self` seems can't be used here.

Reply:
done